test/lib/index-inline.test.ts

```diff
@@ -220,35 +229,53 @@ describe('Test End 2 End - Inline mode', () => {
     expect(mockExit).toHaveBeenCalledWith(0);
   });
 
-  it('Test Inline mode strict mode - no monitored project found - return vulns and 1 exit code', async () => {
-    setTimeout(() => {
-      stdinMock.send(
-        fs
-          .readFileSync(
-            fixturesFolderPath + 'snykTestsOutputs/test-unmonitored-goof.json',
-          )
-          .toString(),
-      );
-      stdinMock.send(null);
-    }, 100);
+  // it('Test Inline mode strict mode - no monitored project found - return vulns and 1 exit code', async () => {
```

Comment:
Why is this commented? we don't need this test anymore?

Reply:
lol, my bad. updated/fixed